packages/storybook/src/ui/molecules/questionnaire-table/questionnaire-table.test.tsx

```diff
@@ -116,4 +116,49 @@ describe('Questionnaire Table', () => {
 
     expect(getTopLeftDataCell()).toHaveTextContent('17-Feb-2022 17:23')
   })
+
+  it('Expands collapsed rows when parent row is clicked', () => {
```

Comment:
Could we add either a third test or a few more assertions to make sure the data disappears again when the row is collapsed? I'd use by `queryBy` for this